Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Eldritch Government

## Changelog:
- Added Eldritch Government:
  - No vassal contracts, taxes, and levies based on opinion.
  - Only rulers of the same religious family can inherit.
  - Dark Authority instead of Crown Authority:
    - Frenzy decays over time so to be efficient they should do their mechanics.
    - High Frenzy gives you skills per stress level and development growth.
  - Get Frenzy on executions.
  - Get Frenzy on mind breaks.
  - Mind breaks can't cause death.
  - Attracts Void Being Knights and Guests with high Skills.
- Nerfed demonic race traits.
- Gave attributes to Nerubian, N'raqi, C'Thrax, Qiraji, Mantid race traits.
- Demonic and Eldritch AI executes prisoners more often.
- Demonic AI tortures prisoners more often.

# Tasks:
- [x] Add unique government bonuses and mention in the government concept
- [x] Add backgrounds


# How to test:
Test gameplay features, tell if they look balanced.

In the Debug mode, right-click Eldritch government rulers to Give and Lose Dark Frenzy fast.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/129772245-54e7262d-ff92-43ed-b554-483e604d5042.png)

</details>

@Warcraft-GoA-Development-Team/english-proofreaders open the [Page](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/600/files?file-filters%5B%5D=.yml), scroll to the bottom to the `.yml` files, check them, suggest changes. If you forgot, here are the [Suggestion Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-to-be-Tester-(For-CK3)#reviewing-code).
